README.rst

```diff
@@ -124,3 +190,4 @@ For more information, see CONTRIBUTING_.
 .. _changelog: http://autoprotocol-python.readthedocs.io/en/latest/changelog.html
 .. _CONTRIBUTING: http://autoprotocol-python.readthedocs.io/en/latest/CONTRIBUTING.html
 .. _documentation: http://autoprotocol-python.readthedocs.org/en/latest/
+.. _devenvsetup: https://strateos.atlassian.net/wiki/spaces/ENG/pages/191660547/Python+environment+setup
```

Comment:
I'm not super keen on linking our own internal documentation in a public repository

Reply:
Ya, @yangchoo mentioned the same. I forgot this is a public repo. Will remove it.